From 3ca0a15ecf19ed313fc9de989ea24f2d98a3a2d7 Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Tue, 26 Jan 2010 11:26:45 +0800
Subject: [PATCH 145/442] ENGR00120052 MX51: Change vpu unit test due to f/w
 upgrade to 1.3.0

Report the failure reason with vpu_DecGetIntialInfo function

Signed-off-by: Sammy He <r62914@freescale.com>
---
 test/mxc_vpu_test/dec.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/test/mxc_vpu_test/dec.c b/test/mxc_vpu_test/dec.c
index d09bf0c..3e68ae7 100644
--- a/test/mxc_vpu_test/dec.c
+++ b/test/mxc_vpu_test/dec.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2004-2009 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright 2004-2010 Freescale Semiconductor, Inc.
  *
  * Copyright (c) 2006, Chips & Media.  All rights reserved.
  */
@@ -1438,7 +1438,8 @@ decoder_parse(struct decode *dec)
 	ret = vpu_DecGetInitialInfo(handle, &initinfo);
 	vpu_DecSetEscSeqInit(handle, 0);
 	if (ret != RETCODE_SUCCESS) {
-		err_msg("vpu_DecGetInitialInfo failed %d\n", ret);
+		err_msg("vpu_DecGetInitialInfo failed, ret:%d, errorcode:%d\n",
+		         ret, initinfo.errorcode);
 		return -1;
 	}
 
-- 
1.8.0

